docs/units/index.rst

```diff
@@ -82,6 +82,12 @@ Units that "cancel out" become a special unit called the
 
     >>> u.m / u.m
     Unit(dimensionless)
+    
+To create a simple :ref:`dimensionless quantity <doc_dimensionless_unit>`, multiply a value by the unscaled dimensionless unit::
```

Comment:
Nitpick, but could you break this line so that it is no more than 80 characters? Otherwise, this looks all OK.

Reply:
Oops, slipped by me.